### PR TITLE
Clarify Comlink.proxyValue(value) example.

### PR DESCRIPTION
**Pull request by gruns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,16 +131,16 @@ around:
...
 ```
 
 ```js
 // worker.js
-Comlink.expose(async function (f) {
-  const result = /* omg so expensive */;
-  /* f is a proxy, as if created by proxy(). So we need to use `await. */
-  await f(result);
+Comlink.expose(async function (callback) {
+  const result = await doSomethingTimeConsuming();
```
Comment (surma) [inline README.md]: I’d prefer no `await` here as it is perfectly okay (and even one of the motivations) to use blocking code in workers

**Comment by surma** [comment]
Thanks for improving the example, @gruns! I really appreciate it. However, I do need to you to sign the CLA. Please follow the instructions @googlebot posted :)

**Comment by gruns** [comment]
Removed the await before `doSomethingTimeConsuming()`, as requested.

I also explicitly added `proxyValue()` to the example for clarification. This clarification is helpful; I had to read comlink's source to figure out`proxyValue()`'s proper use.

What else can I tweak or improve? I'll also sign the CLA shortly.

Thanks so much @surma.

**Comment by surma** [comment]
This LGTM. Just waiting for the CLA :)

**Comment by gruns** [comment]
@surma CLA signed.

**Comment by surma** [comment]
Sorry for the long turn around on this, I was on vacation. Would you mind doing me one last favor and rebasing against master?

**Comment by gruns** [comment]
Squashed and rebased.

**Comment by surma** [comment]
A+. Thanks!!

**Pull request merged**